Documentation/faq.md

```diff
@@ -0,0 +1,17 @@
+## Frequently Asked Questions (FAQ)
+
+### Configuration
+
+#### What is the difference between `advertise-urls` and `listen-urls`?
+
+`listen-urls` specifies the addresses etcd server binds to. etcd server accepts incoming connections from the interfaces it binds to. To listen on a port for all interfaces, specify `0.0.0.0` as listen IP address.
+
+`advertise-urls` specifies the addresses etcd server advertises to its clients or other etcd members. The advertise addresses must be reachable from the remote machines. Do not advertise addresses like `localhost` or `0.0.0.0` for a production setup. These addresses are not reachable from remote machines.
```

Comment:
`advertise-urls` specifies the addresses etcd clients or other etcd members should use to contact the etcd server. The advertise addresses must be reachable from the remote machines. Do not advertise addresses like `localhost` or `0.0.0.0` for a production setup since these addresses are unreachable from remote machines.